source/jsonnet/northern-ireland/individual/blocks/personal-details/term_time_country_outside_uk.jsonnet

```diff
@@ -0,0 +1,44 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local question(title) = {
+  id: 'term-time-country-outside-uk-question',
+  title: title,
+  type: 'General',
+  answers: [
+    {
+      id: 'term-time-country-outside-uk-answer',
+      label: 'Current name of country',
+      description: 'Enter your own answer or select from suggestions',
+      mandatory: false,
+      type: 'TextField',
+    },
+  ],
+};
+
+{
+  type: 'Question',
+  id: 'term-time-country-outside-uk',
+  question_variants: [
+    {
+      question: question('During term time, in which country outside the UK do you usually live?'),
+      when: [rules.isNotProxy],
+    },
+    {
+      question: question({
+        text: 'During term time, in which country outside the UK does <em>{person_name}</em> usually live?',
+        placeholders: [
+          placeholders.personName,
+        ],
+      }),
+      when: [rules.isProxy],
+    },
```

Comment:
This is passing it inline, the `term-time-address` is passing vars, which is inconsistent. Given that we have a card to make all these consistent, not a blocker but would ease future work.